components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -1041,6 +1057,71 @@ class BraveRewardsBrowserTest :
             AsWeakPtr()));
   }
 
+  void OpenMonthlyStatements() {
+    content::EvalJsResult monthlyStatementClickResult = EvalJs(contents(),
+        "document.querySelector(\"[data-test-id='monthlyStatementButton']\")"
+        ".click();"
+        "new Promise((resolve) => {"
+        "var count = 10;"
+        "var interval = setInterval(function() {"
+        "  if (count == 0) {"
+        "    clearInterval(interval);"
+        "    resolve(false);"
+        "  } else {"
+        "    count -= 1;"
+        "  }"
+        "  if (document.getElementById('modal')) {"
+        "    clearInterval(interval);"
+        "    resolve(document.getElementById('modal') != null);"
+        "  }"
+        "}, 500);});",
+        content::EXECUTE_SCRIPT_DEFAULT_OPTIONS,
+        content::ISOLATED_WORLD_ID_CONTENT_END);
+    ASSERT_TRUE(monthlyStatementClickResult.ExtractBool());
+    monthly_statement_opened_ = true;
+    if (wait_for_monthly_statement_to_open_loop_) {
+      wait_for_monthly_statement_to_open_loop_->Quit();
+    }
+  }
+
+  void OpenMonthlyAutoContributeReportTab() {
+    content::EvalJsResult acTabClickResult = EvalJs(contents(),
+        "document.querySelector(\"[data-test-id='cont3']\")"
+        ".click();"
+        "new Promise((resolve) => {"
+        "var count = 10;"
+        "var interval = setInterval(function() {"
+        "  if (count == 0) {"
+        "    clearInterval(interval);"
+        "    resolve(false);"
+        "  } else {"
+        "    count -= 1;"
+        "  }"
+        "  if (document.querySelector(\"[data-test-id='acTable']\")) {"
+        "    clearInterval(interval);"
+        "    resolve(document.querySelector(\"[data-test-id='acTable']\")"
+        "         != null);"
+        "  }"
+        "}, 500);});",
+        content::EXECUTE_SCRIPT_DEFAULT_OPTIONS,
+        content::ISOLATED_WORLD_ID_CONTENT_END);
+    ASSERT_TRUE(acTabClickResult.ExtractBool());
+    ac_tab_opened_ = true;
+    if (wait_for_ac_tab_to_open_) {
+      wait_for_ac_tab_to_open_->Quit();
+    }
+  }
+
+  int32_t GetTransactionEntryCount() {
+    content::EvalJsResult acTableCountResult = EvalJs(contents(),
+        "document.querySelector(\"[data-test-id='txTable']\")"
+        ".getElementsByTagName('tbody')[0].childNodes.length",
+        content::EXECUTE_SCRIPT_DEFAULT_OPTIONS,
+        content::ISOLATED_WORLD_ID_CONTENT_END);
+    LOG(ERROR) << "=====table rows: " << acTableCountResult.ExtractInt();
```

Comment:
Is this a leftover debug statement or intentional?

Reply:
removed.